django/contrib/redirects/migrations/0001_initial.py

```diff
@@ -27,10 +20,15 @@ class Migration(migrations.Migration):
                     help_text="This can be either an absolute path (as above) or a full URL starting with 'http://'.",
                     max_length=200, verbose_name='redirect to', blank=True
                 )),
+                # We need to add `domain` field to migrate sites references
```

Comment:
Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).